src/java/com/phenix/pct/AnalyticsBuildListener.java

```diff
@@ -0,0 +1,90 @@
+package com.phenix.pct;
+
+import java.io.DataOutputStream;
+import java.io.IOException;
+import java.net.HttpURLConnection;
+import java.net.URL;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.ResourceBundle;
+import java.util.Map.Entry;
+
+import org.apache.tools.ant.BuildEvent;
+import org.apache.tools.ant.BuildListener;
+import org.apache.tools.ant.Task;
+
+public class AnalyticsBuildListener implements BuildListener {
+    private static final Map<String, Integer> taskCount = new HashMap<>();
+
+    @Override
+    public void taskFinished(BuildEvent event) {
+        Task task = event.getTask();
+        if (task == null) {
+            return;
+        }
+        String type = task.getTaskType();
+        if (type == null) {
+            return;
+        }
+        if (type.toLowerCase().startsWith("pct") || "RestGen".equalsIgnoreCase(type)
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png 'Severity: CRITICAL') Reduce the number of conditional operators (6) used in the expression (maximum allowed 3). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.riverside-software.fr/coding_rules#rule_key=squid%3AS1067)